models/modelling/commissioning/demographics/int_person_pds_population_flags.sql

```
@@ -13,26 +17,26 @@ select
         pds.record_registered_end_date is null
-    ) as flag_current_ncl_registered,
+    ) as flag_current_registered,
     pds.record_registered_start_date as record_registered_start_date,
 
-    --Flag for current NCL Resident population
+    -- Flag for current WNL resident population
     (
-        --LSOA 21 is in NCL
-        geo.resident_flag = 'NCL' and
+        --LSOA 21 is in NCL or NWL
+        geo.resident_flag in ('NCL', 'NWL') and
```

SQLFluff (4.2.2)

[error] Found trailing binary operator. Expected only leading near line breaks.

(LT03)